docs/design/StratisSoftwareDesign.lyx

```diff
@@ -6965,8 +6965,30 @@ LUKS2 tokens contain the necessary information to describe how to unlock
  the device using a passphrase stored in the kernel keyring and another
  Stratis-defined token that contains activation information required by
  Stratis.
- Token indexes are statically assigned for each token so that new tokens
- may be added without breaking changes to the existing tokens.
+ Token IDs are statically assigned for each token for a number of reasons:
+\end_layout
+
+\begin_layout Itemize
+Using static IDs, stratisd no longer has to do a scan of all of the tokens
```

Comment:
You'll want to leave out the "no longer"...it never did need to do this in any release.